packages/prompts/src/text.ts

```diff
@@ -20,7 +20,9 @@ export const text = (opts: TextOptions) => {
 		signal: opts.signal,
 		input: opts.input,
 		render() {
-			const title = `${color.gray(S_BAR)}\n${symbol(this.state)}  ${opts.message}\n`;
+			const withBorder = opts.withBorder !== false;
+			const titlePrefix = withBorder ? `${color.gray(S_BAR)}\n${symbol(this.state)}  ` : '';
```

Comment:
IIUC this means no symbols prefixing prompts when disabling the border? It’s personal preference I guess, but I’d still expect some kind of leading marker even when disabling borders/guides. These markers can be helpful indicators of state.

For example, to mirror something a bit like what `prompts` does, state is indicated via something like
```
? What is your name?
  Placeholder
```
And then post-confirm
```
✔︎ What is your name?
  Chris
```

These could definitely just be the standard clack symbols by default but the left border/guide seems a distinct UI element from these status indicators to me.